features/product/viewing_product_in_admin_panel/viewing_product_catalog_price_history_for_a_simple_product.feature

```diff
@@ -0,0 +1,39 @@
+@viewing_products
+Feature: Seeing catalog price history for a simple product
+    In order to be aware of historical simple product prices
+    As an Administrator
+    I want to browse the catalog price history of a simple product
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Wyborowa Vodka" priced at "$1000.00" in "United States" channel
+        And there is disabled catalog promotion named "Winter sale"
+        And the catalog promotion "Winter sale" is available in "United States"
+        And it applies on "Wyborowa Vodka" product
+        And it reduces price by "90%"
+        And I am logged in as an administrator
+
+    @todo
+    Scenario: Seeing the catalog price history of a simple product
+        Given the "Winter sale" catalog promotion is enabled
+        When I disable "Winter sale" catalog promotion
+        And I access "Wyborowa Vodka" product
+        And I go to the product price history
+        Then I should see 3 log entries in the catalog price history for "Wyborowa Vodka" variant
+        And there should be a log entry on the 2nd position with the "$1000.00" selling price, "$1000.00" original price and datetime of the price change
+        And there should be a log entry on the 2nd position with the "$100.00" selling price, "$1000.00" original price and datetime of the price change
+        And there should be a log entry on the 3rd position with the "$1000.00" selling price, "$1000.00" original price and datetime of the price change
+
+    @todo
+    Scenario: Seeing the catalog price history of a simple product with a price different than an original price
+        Given "Wyborowa Vodka" variant is originally priced at "$1200.00" in "United States" channel
```

Comment:
This step confuses me along with the product creation in the background. I expected in the 39 line that:
```
And there should be a log entry on the 4rd position with the "$1000.00" selling price, "$1000.00" original price and datetime of the price change
```
as this is the first action related to product prices.

Here we set only the catalog price and the original price is also logged:
https://github.com/Sylius/PriceHistoryPlugin/pull/5/files#diff-53274ef254d391f8fd5a185c3c9d6c64912752970ed51b4c83a9d9ecd52a7cd2R25-R34

Reply:
Right! Should be 1000, 1000, and then 1000, 1200. However, I would like to test if a different price and the original price are logged as the first entry so I probably should create such a product in the background.

Reply:
Already tested in https://github.com/Sylius/PriceHistoryPlugin/pull/5/files#diff-53274ef254d391f8fd5a185c3c9d6c64912752970ed51b4c83a9d9ecd52a7cd2R22 so I did this change in line 39 as you suggested :+1: